client/app/bundles/course/assessment/submission/pages/SubmissionsIndex/SubmissionsTable.jsx

```diff
@@ -107,13 +107,13 @@ export default class SubmissionsTable extends React.Component {
             {submission.courseStudent.name}
           </a>
         </TableRowColumn>
-        <TableRowColumn style={styles.tableCell}>
+        <TableRowColumn style={{ ...styles.tableCell, textAlign: 'center' }}>
```

Comment:
You can define a new style called `centralisedTableCell` which merges the `textalign` with the `tableCell` styles. I think that will be cleaner.